### ISSUE TITLE
chartime

### ISSUE THREAD

**Issue by liangdu** [sebastianGehrmann/phenotyping#2]
I am trying to use the annotation.cvs with MIMIC-III 's noteevents table. 

But seems the identifier chart_time is in different formation, not TIMESTAMP type as used by MIMIC-III chart time. 

How to convert it ? I assume that I need the three identifier combined to get the text for the 1610 records ?

Thanks in advance.

**Comment by sebastianGehrmann** [comment]
This is an unfortunate side effect of the annotation software we used. You have two options: 
1) Get the text data from https://physionet.org/works/AnnotatedClinicalTextsACTfromMIMIC/ - just use the same username and password that you used for MIMIC
2) Follow the instructions here: https://ektar.github.io/2016/mimic-iii-note-matching.html

**Comment by liangdu** [comment]
Thanks for the reply. Seems the data in  https://physionet.org/works/AnnotatedClinicalTextsACTfromMIMIC/  is not available. 

I will try option 2.

**Comment by sebastianGehrmann** [comment]
What exactly do you mean not available? Are you having trouble accessing it?

**Comment by sebastianGehrmann** [comment]
@EdwardMoseley resolved the issue, does it work now?

**Comment by liangdu** [comment]
Yes. The dataset is not available. I have my mimic-iii login.  I emailed
Edward but get no response .

**Comment by EdwardMoseley** [comment]
Please apply for access here: https://physionet.org/works/AnnotatedClinicalTextsACTfromMIMIC/

**Comment by liangdu** [comment]
Thanks. Applied !
